xml/art_migration.xml

```diff
@@ -576,24 +584,14 @@ mig-cp1-neut-m1-mgmt</screen>
     <itemizedlist>
      <listitem>
       <para>
-       On each &slsa; compute node, remove outdated repositories that point
-       to the old deployer. To do this, run <command>zypper lr -u</command> and
-       remove listings referring to the original deployer's IP address
-       using <command>sudo zypper rr <replaceable>REPO_NAME</replaceable></command>.
-       Note that the repositories <systemitem>SLES-OS</systemitem> and
-       <systemitem>SLES-SDK</systemitem> should never be removed, as they are
-       system entries that will automatically be updated to point to the new
-       seed node during installation.
-      </para>
-      <para>
-       If your compute nodes are kept up to date using the &suse; Updates
-       repository via an SMT server, you should see a repository for this
-       purpose defined on your compute nodes. For proper behavior during the
-       upgrade, your repository names should be <systemitem>SLES12-SP3-Pool</systemitem>
-       and <systemitem>SLES12-SP3-Updates</systemitem>. If necessary, you can
-       rename any existing repository to match these names by running:
+       On each SLES compute node, remove all the repositories except for
```

Comment:
The title above this should be Update repository references of Compute Node**s**

Reply:
not sure Node is capitalized, if it is not there are other places as well